specs/monitoring_integration.adoc

```diff
@@ -0,0 +1,390 @@
+= Spec - Introducing a new component monitoring-integration
+
+A new component called Monitoring-Integration is introduced which integrates Grafana
+with tendrl core.
+
+
+== Problem description
+
+The new monitoring-integration component would primarily focus on dynamic
+configuration of grafana, based on the topology changes in etcd. Some of these
+responsibilities would be:
+
+* Monitoring-integration should store predefined json’s for dashboards, panels and alerts.
```

Comment:
Monitoring-integration should create default dashboards in grafana using predefined json.
The dashboard json contains details regarding panels.,metrics and alerts.